Makefile

```diff
@@ -123,6 +129,43 @@ kind-load-test-artifacts: $(KIND) ## Load the e2e testdata container images into
 	$(KIND) load docker-image localhost/testdata/bundles/plain-v0/plain:v0.1.0 --name $(KIND_CLUSTER_NAME)
 	$(KIND) load docker-image localhost/testdata/catalogs/test-catalog:e2e --name $(KIND_CLUSTER_NAME)
 
+.PHONY: opm
+OPM = ./bin/opm
+OPM_VERSION = v1.28.0
+opm: ## Download opm locally if necessary.
```

Comment:
Preferably if we're going to be using the opm binary in operator-controller then we should add it with [`bingo`](https://github.com/bwplotka/bingo) as we've done with the other binary dependencies. You should be able to add that with:  
```
$ bingo get github.com/operator-framework/operator-registry/cmd/opm@v1.28.0
```
The same applies to `operator-sdk` down below, you should be able to add that to the project as well with:
```
$ bingo get github.com/operator-framework/operator-sdk/cmd/operator-sdk@v1.31.0
```